client/wayland/ibuswaylandim.c

```diff
@@ -1174,6 +1457,7 @@ _create_input_context_done (GObject      *object,
                           G_CALLBACK (_context_forward_key_event_cb),
                           wlim);
 
+#if 0
         g_signal_connect (priv->ibuscontext, "update-preedit-text-with-mode",
```

Comment:
`_context_update_preedit_text_cb` is calling `zwp_input_method_v2_set_preedit_string` so I assume this should now be active. Or is this not supposed to be working yet?

Reply:
Yeah, I commented out here to debug an `input_popup_surface` SEGV. Now I fixed the SEGV and revert this.